Remotion/Web/Development.WebTesting/Web.Development.WebTesting.csproj

```diff
@@ -22,10 +22,10 @@
     <ProjectReference Include="..\Contracts.DiagnosticMetadata\Web.Contracts.DiagnosticMetadata.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fork.Coypu" Version="3.3.0" />
+    <PackageReference Include="Fork.Coypu" Version="3.4.0-alpha.1" />
     <PackageReference Include="log4net" Version="2.0.15" />
-    <PackageReference Include="Selenium.Support" Version="4.8.1" />
-    <PackageReference Include="Selenium.WebDriver" Version="4.8.1" />
+    <PackageReference Include="Selenium.Support" Version="4.11.0" />
```

Comment:
please don't forget linking the release notes into the jira ticket so downstream projects have it easier finding the changes.